NLog.Targets.KafkaAppender/KafkaProducerAsync.cs

```diff
@@ -17,18 +15,10 @@ public override void Produce(string topic, string data)
 
         private async Task ProduceAsync(string topic, string data)
         {
-            try
+            await Producer.ProduceAsync(topic, new Message<Null, string>
```

Comment:
Since you no longer catching exceptions, then it can be simplified (Removing async + await)
```c#
private Task ProduceAsync(string topic, string data)
{
    return Producer.ProduceAsync(topic, new Message<Null, string>{
                Value = data
            });
}
```

Reply:
It's like "fire and forget". I think it's a good case for this scenario
Thanks!